stglib/wxt.py

```diff
@@ -84,21 +84,34 @@ def cdf_to_nc(cdf_filename):
     # Rename variables to CF compliant names
     ds = ds_rename_vars(ds)
 
-    # Convert data types to float 32
-    # ds = ds_convertfloat(ds)
+    # force WD vars to floats
+    if "WD_min" in ds:
+        ds["WD_min"] = ds["WD_min"].astype(float)
+    if "WD_410" in ds:
+        ds["WD_410"] = ds["WD_410"].astype(float)
+    if "WD_gust" in ds:
+        ds["WD_gust"] = ds["WD_gust"].astype(float)
 
     # If sensor wasn't pointing to magnetic north, apply offset to direction
     if "dir_offset" in ds.attrs:
         if "WD_min" in ds:
-            ds["WD_min"].values = ds["WD_min"].values + ds.attrs["dir_offset"]
+            ds["WD_min"].values = ds["WD_min"].values + ds.attrs["dir_offset"].astype(
```

Comment:
I am pretty sure all these `.values` are unnecessary. Have you tried without? Mostly a readability thing

Reply:
Should be all set now. removed .values and ran test on 2 datasets and test_scripts.py. all passed.